templates/install/project.md

```diff
@@ -104,22 +104,30 @@ Also, you can get the Lean documentation inside VS Code using
 inside the text field that appears, type "lean doc" and hit <kbd>Enter</kbd>.
 Then click "Mathematics in Lean" or "Theorem Proving in Lean" and enjoy.
 
-## Hosting your project on GitHub
+## Updating Mathlib in your project
 
-Your project is already a git repository, and as it grows,
-you may want to host it on [GitHub](https://guides.github.com/activities/hello-world/).
-If you take this step, the community offers some
-[GitHub Actions scripts](../ci.html) that could help manage your repository.
-But don't worry if you don't know what this means.
-It's not necessary for using Lean.
+If you have a project depending on Mathlib, and you want to update to the latest version of Lean and Mathlib, you have to do two steps:
+* Update the `lean-toolchain` file in your repository to the latest version. You can do this automatically by running the following in the root directory of your repository.
+```
+curl https://raw.githubusercontent.com/leanprover-community/mathlib4/master/lean-toolchain > lean-toolchain
+```
+* run `lake update`. This will update Lean, Mathlib and download the new Mathlib cache for you.
+
+You should then check whether all files in your repository still compile. The most convenient way to do this is to have a root file that imports all files in your repository. You can do this by running the following (replace `MyProject` twice by your project name):
+```
+find MyProject -name "*.lean" | env LC_ALL=C sort | sed 's/\.lean//;s,/,.,g;s/^/import /' > MyProject.lean
```

Comment:
I thought now it's preferable to use `lake exe mk_all`? See [Zulip](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/automatically.20generate.20ProjectName.2Elean).

Reply:
Thanks for the link. I have issues with `mk_all`, but if that works reliably, we can adapt to that.

Reply:
It seems that there is an error with this script on Windows. I suggest we use the current instructions for now, and we can simplify them once that issue is fixed.

Reply:
I mention `lake exe mk_all` now with the caveat that it doesn't work on Windows.